usr/local/bin/install-ukca-data

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Install UKCA input data:
+# 1) wget tar file from JASMIN
+# 2) Extract input files from tar file
+# 3) Run md5sum on files to check for data corruption
+# 4) Copy to umdir
+# 5) Clean-up
+
+if [ "$#" -gt 1 ]; then
+  echo "Usage: $(basename $0) [version]"
+  exit 1
+fi
+
+ver=CUR
+if [[ -n "$1" ]]; then
+  ver=$1
+fi
+
+tmpdir=$(mktemp -d)
+cd $tmpdir
+
+tarfile=ukca_ancils
+
+echo "Extracting input data archive from JASMIN UKCA GWS"
+wget -q --show-progress http://gws-access.ceda.ac.uk/public/ukca/${tarfile}_v${ver}.tgz
+if [ $? -ne 0 ]; then
+  echo "Error downloading UKCA data"
+  exit 1
+fi
+
+tar -xzf ${tarfile}_v${ver}.tgz
+
+# now md5 sum the files to make sure there has been no corruption
+echo "Checking files..."
+md5sum --status -c ukca_ancils.md5
+if [ $? -ne 0 ]; then
+  echo "Error extracting UKCA files"
+  exit 1
+fi
+
+# If a central install (using rose-stem) has been attempted before the ancils
+# have been set up, this could have left a dead symlink which needs removing:
+if [[ -h "$HOME/umdir/ancil" ]]; then
+  rm -f $HOME/umdir/ancil
+fi
+
+mkdir -p $HOME/umdir
+cp -r $tarfile/* $HOME/umdir
+
+# Clean up
+rm -rf $tmpdir
+
+echo
+echo "UKCA Inputs have been installed to $HOME/umdir/ancil"
+echo "and $HOME/umdir/standard_jobs"
```

Comment:
You mean $HOME/umdir/standard_jobs? If someone has run the install-um-data script then they will have had something put there, but otherwise not I believe.

When writing this script I basically copied what was done for the install-um-data script. In that file there is a check for $HOME/umdir/ancil being a symbolic link (line 31 of install-um-data) which I mirrored in line 45 of install-ukca-data. The comment there is that rose stem could be making symbolic links to $UMDIR/ancil which need to be cleaned-up prior to running this script. 

Would you like me to put an equivalent line for $HOME/umdir/standard_jobs ? I doubt that this would be needed in practice if it isn't needed for install-um-data.